modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/manager/DefaultExtensionManagerTestCase.java

```diff
@@ -80,7 +80,7 @@
 
   private static final String MULESOFT = "MuleSoft";
   private static final String OTHER_VENDOR = "OtherVendor";
-  private static final XmlDslModel XML_DSL_MODEL = XmlDslModel.builder().setNamespace("extension-namespace").build();
+  private static final XmlDslModel XML_DSL_MODEL = XmlDslModel.builder().setPrefix("extension-namespace").build();
```

Comment:
not sure if you didn't see the commit [fixes after MGs review](https://github.com/mulesoft/mule/pull/4180/commits/01c58861231d7e49389197e63138a30452b2d9a2), but they weren't ignored